pack/bkw.json

```diff
@@ -119,6 +265,64 @@
     "traits": "Preparation. Skill.",
     "type_code": "upgrade"
   },
+  {
+    "code": "08018",
+    "cost": 1,
+    "deck_limit": 3,
+    "faction_code": "justice",
+    "name": "Spycraft",
+    "pack_code": "bkw",
+    "position": 18,
+    "quantity": 3,
+    "resource_physical": 1,
+    "text": "Play only if you control a [[Spy]] character.\n<b>Interrupt</b>: When you reveal an encounter card, discard Spycraft → cancel the effects of that card discard it. Then, reveal another card from the encounter deck.",
+    "traits": "Preparation. Skill.",
+    "type_code": "upgrade"
+  },
+  {
+    "code": "08023",
+    "cost": 3,
+    "deck_limit": 1,
+		"is_unique": true,
+    "faction_code": "basic",
+    "name": "Quincarrier",
+    "pack_code": "bkw",
+    "position": 17,
```

Comment:
Starting here it looks like all the positions are set to '17' instead of matching what's on the pack